index.html

```diff
@@ -5,20 +5,35 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="styles.css"/>
+  <link rel="stylesheet" href="queries.css"/>
   <title>Portfolio</title>
 </head>
 <body>
   <main class="container">
   
     <header class="header">
-      <nav>
+      <nav class="desktop-navbar">
+        <div class="desktop-nav">
+          <a href="#" target="_parent" class="logo">Tanveer Ahmad</a>          
+        </div>
+        <div class="nav-links">  
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
```

Comment:
Great job! but it's required to follow the original Figma design, so I please make sure to use the same color for your navbar menu links. 

## Original
![image](https://user-images.githubusercontent.com/34779928/200074543-d6fcb32f-d102-4100-bc0d-cdd5899631df.png)

## Yours
![image](https://user-images.githubusercontent.com/34779928/200074621-64ac9d8e-34c1-497c-9548-bb39e38835a1.png)